stdlib/public/runtime/RefCount.cpp

```diff
@@ -173,7 +173,7 @@ void RefCounts<SideTableRefCountBits>::incrementUnownedSlow(uint32_t n) {
   swift_abortUnownedRetainOverflow();
 }
   
-SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_API
```

Comment:
I don't think you can make this change, it is technically ABI breaking. I think the thing to do is to mark the definition in the header file as `SWIFT_CC(swift)` to match what was exported.

Reply:
ok; I can change it back and adjust the header - it didn't seem to be used anywhere so I wasn't sure the intent